python/py-ukpostcodeparser/Portfile

```
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-ukpostcodeparser
+set real_name       UkPostcodeParser
+version             1.1.2
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainers
+
+description         UK Postcode parser
+long_description    ${description}
```

```suggestion
long_description    {*}${description}
```

When cloning the description you need to add this to correctly expand things.

---

Done

---

that might be necessary in certain cases, but here with a "simple" `description` I am pretty sure there is no actually difference.... or?